lib/active_merchant/billing/gateways/commerce_hub.rb

```diff
@@ -223,9 +223,22 @@ def add_payment_token(source, payment, options)
         end
       end
 
+      def add_decypted_wallet(source, payment, options)
```

Comment:
small typo, I think it would be add_decrypted_wallet

Reply:
yes, thanks